internal/component/database_observability/postgres/collector/connection_info.go

```diff
@@ -34,13 +38,14 @@ func NewConnectionInfo(args ConnectionInfoArguments) (*ConnectionInfo, error) {
 		Namespace: "database_observability",
 		Name:      "connection_info",
 		Help:      "Information about the connection",
-	}, []string{"provider_name", "provider_region", "db_instance_identifier", "engine"})
+	}, []string{"provider_name", "provider_region", "db_instance_identifier", "engine", "engine_version", "engine_version_suffix"})
 
 	args.Registry.MustRegister(infoMetric)
 
 	return &ConnectionInfo{
 		DSN:        args.DSN,
 		Registry:   args.Registry,
+		DBVersion:  args.DBVersion,
```

Comment:
should we use the same `engine terminology`?

Reply:
Fair play. Updated that as well. In MySQL we might consider renaming that in the explain plan schema as well.